.github/workflows/update-content-scope.yml

```diff
@@ -32,27 +32,37 @@ jobs:
         run: ncu @duckduckgo/content-scope-scripts --errorLevel 2
 
       - name: Update content scope scripts
-        if: ${{ steps.update-check.outcome == 'failure' }}
+        if: ${{ steps.update-check.outputs.outcome == 'failure' }}
```

Comment:
`output` is not correct, it should only be `outcome`, same for all the other steps in both workflows when you refer to the whole step failing.

Reply:
It’s failing because there is no PR to modify. You executed the action manually, not via a PR. The if clause is correct

Reply:
This workflow is executed nightly or manually, I don't see why we would need a PR to be able to execute it? The if clause can never work because outcome is not part of the outputs of a step, if you check the diff you will see it didn't have that before.

Reply:
when you are right, you are right